docs/kratos/hooks/01_configure-hooks.mdx

```diff
@@ -0,0 +1,373 @@
+| :------ | :-------: | :--------- |
+| `url`   | Yes | URL called by the webhook. |
+| `method`| Yes | HTTP method used to send a request to the webhook URL. |
+| `body`  | No | Base64-encoded Jsonnet template used to render the payload. Ignored when using an HTTP method that doesn't allow sending body.|
+| `auth`  | No | Webhook authentication mechanism configuration. |
+
+  </TabItem>
+  <TabItem value="selfhosted" label="Self-Hosted Ory Kratos Config" default>
+
+```yaml
+selfservice:
+  flows:
+    login:
+      before:
+        hooks:
+          - hook: web_hook # To use webhooks, you must set 'hook' to 'web_hook'
+            config:
+              url: https://test.hook.site.sh/before_login_hook # Webhook URL.
+              method: POST # HTTP method used to send request to the webhook URL.
+              body: file:///path/of/my/jsonnet/file # Jsonnet template URI used to render payload.
+              # Ignored when using an HTTP method that doesn't allow sending body. OPTIONAL
+              auth:
+                type: # Can be one of 'basic_auth' or 'api_key'
+                config: # Additional auth config for the hook. Read next section for details.
+```
+
+| Setting | Mandatory | Description
+| :------ | :-------: | :--------- |
+| `url`   | Yes | URL called by the webhook. |
+| `method`| Yes | HTTP method used to send a request to the webhook URL. |
+| `body`  | No | Jsonnet template URI used to render payload. Use `file://path/to/body.jsonnet` when referring to local files. Ignored when using an HTTP method that doesn't allow sending body.|
+| `auth`  | No | Webhook authentication mechanism configuration. |
+
+  </TabItem>
+</Tabs>
+````
+
+:::info
+
+Webhooks can't control the execution of the flow by patching/updating particular properties of flow-specific objects on webhook
+completion.
+
+The execution of the flow can be canceled if the call to the webhook endpoint fails either due to a network error or if the
+endpoint returns an HTTP code `4xx` or `5xx` response.
+
+:::
+
+#### Request body
+
+Webhooks bind the `flow`, as well as request headers (`request_headers`), request method (`request_method`), and the request URL
+(`request_url`) of the flow into the Jsonnet template for all methods and execution paths (before and after). For the `after`
+execution path of all flows, it binds the `identity` object into the Jsonnet template as well. These objects are available through
+a `ctx` object.
+
+To send `{ user_id: <some-id> }` in the request body, create the following the Jsonnet template:
+
+```jsonnet
+function(ctx) { user_id: ctx.identity.id }
+```
+
+#### Authentication and authorization in webhooks
+
+For `auth` following mechanisms are supported:
+
+- API key authentication. Type must be set to `api_key`. All properties are mandatory.
+
+  ```yaml
+  - hook: web_hook
+    # <other configuration keys>
+    config:
+      auth:
+        type: api_key
+        config:
+          name: Some-Name
```

Comment:
I guess having a "real" name that is used to authenticate may say a little more about this property

```suggestion
          name: Authorization
```